docker/nginx/default.conf

```diff
@@ -1,13 +1,26 @@
 server {
-  listen 80 default_server;
+  listen 80 default_server reuseport;
 
   keepalive_timeout 70;
 
   index index.php;
 
-  location / {
-    root /usr/share/nginx/html;
-    try_files $uri /index.php$is_args$args;
+  root /usr/share/nginx/html;
+  try_files $uri /index.php$is_args$args;
+
+  # security headers
+  add_header X-XSS-Protection          "1; mode=block" always;
+  add_header X-Content-Type-Options    "nosniff" always;
+  add_header Referrer-Policy           "no-referrer-when-downgrade" always;
+  add_header Content-Security-Policy   "default-src 'self' http: https: ws: wss: data: blob: 'unsafe-inline'; frame-ancestors 'self';" always;
+  add_header Permissions-Policy        "interest-cohort=()" always;
```

Comment:
https://www.permissionspolicy.com/

I think that `interest-cohort` has been deprecated/removed. I skimmed the Further Reading links and didn't find anything about it. Also, the popup explanation on the main link says that it is obsolete.

IMO remove this.